docs/SplunkOperatorUpgrade.md

```diff
@@ -145,6 +148,7 @@ imagePullPolicy: IfNotPresent
 
 This is an example of the process followed by the Splunk Operator if the operator version is upgraded and a later Splunk Enterprise Docker image is available:
 ​
+
```

Comment:
Nit: The numbering here seems incorrect. Point #2 is missing. Can we correct this?